### PR TITLE
Fix `TimeDelta` warning

### PR DESCRIPTION
**Pull request by WilliamJamieson**
Fixes a warning raised when using astropy develop version

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -98,6 +98,11 @@ commands=
     bandit -c bandit.yaml -r .
 
 [testenv:warnings]
+deps =
+    git+https://github.com/numpy/numpy
```
Comment (pllim) [inline tox.ini:102]: There is "nightly" (weekly) numpy dev you can use. See `astropy` setting.

**Comment by pllim** [comment]
If this works for you, seems fine to me. I am not familiar with your test to comment.

**Review by perrygreenfield — COMMENTED** [review]
LGTM

**Review by perrygreenfield — APPROVED** [review]
LGTM

**Pull request merged**